### PR TITLE
feat(docker): Add CloudOps Dockerfile & CircleCI build instructions

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

File: Dockerfile-build [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+WORKDIR /app
+
+# S3 bucket in Cloud Services prod IAM
+ADD https://s3.amazonaws.com/dumb-init-dist/v1.2.0/dumb-init_1.2.0_amd64 /usr/local/bin/dumb-init
+RUN chmod +x /usr/local/bin/dumb-init
+ENTRYPOINT ["/usr/local/bin/dumb-init", "--"]
+
+USER app
+
+COPY package.json package.json
```
Comment (vladikoff) [inline Dockerfile-build]: @jbuck what is this for? do shrinkwrap files also need to be copied here?

**Comment by jbuck** [comment]
Depends on https://github.com/mozilla/fxa-content-server/pull/4621

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fb19bf72917517c2fa5be27dd969a741fc6fc4e8
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: fec0d1d213ea1415fcf361ba4301f436839c2ea5
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

File: package.json [hunk 2]
```diff
@@ -3,7 +3,8 @@
   "version": "0.78.0",
   "description": "Firefox Accounts Content Server",
   "scripts": {
-    "postinstall": "bower update --config.interactive=false -s && scripts/download_l10n.sh",
+    "build-production": "grunt build",
+    "postinstall": "bower update --allow-root && scripts/download_l10n.sh",
```
Comment (vladikoff) [inline package.json]: `"bower update --allow-root` 👎

File: Dockerfile-build [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+COPY bower.json bower.json
+COPY npm-shrinkwrap.json npm-shrinkwrap.json
+COPY package.json package.json
+COPY scripts/download_l10n.sh scripts/download_l10n.sh
+
+RUN npm install --production --unsafe-perm && rm -rf ~/.npm /tmp/*
+
+COPY . /app
+
+RUN npm run build-production --unsafe-perm
```
Comment (vladikoff) [inline Dockerfile-build]: `--unsafe-perm`  I'm really not a fan of these :(

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fb19bf72917517c2fa5be27dd969a741fc6fc4e8
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: fec0d1d213ea1415fcf361ba4301f436839c2ea5
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: a4073387f7d30f6a76969961661a267917043221
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

File: Dockerfile-build [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+WORKDIR /app
+
+# S3 bucket in Cloud Services prod IAM
+ADD https://s3.amazonaws.com/dumb-init-dist/v1.2.0/dumb-init_1.2.0_amd64 /usr/local/bin/dumb-init
+RUN chmod +x /usr/local/bin/dumb-init
+ENTRYPOINT ["/usr/local/bin/dumb-init", "--"]
+
+RUN apk add --no-cache git
+
+COPY .bowerrc .bowerrc
```
Comment (vladikoff) [inline Dockerfile-build]: 👍

**Pull request merged**